starter-code/src/viking.js

```diff
@@ -1,11 +1,84 @@
 // Soldier
-function Soldier() {}
+function Soldier(healthArg, strengthArg) {
+  this.health=healthArg;
+  this.strength =strengthArg;
+  this.attack=function() {
+    return this.strength;
+  }
+  this.receiveDamage=function(damage){
+    this.health=this.health-50;
```

Comment:
Sacar el receiveDamage y el attack como métodos del prototype y a la salud restarle el argumento que recibe la función damage
`Soldier.prototype.receiveDamage = function(damage) {
  this.health -= damage
}`